app/_src/policies/introduction.md

```diff
@@ -1,6 +1,650 @@
 ---
 title: Policies
 ---
+{% if_version gte:2.9.x %}
+## What is a policy?
+
+A policy is a set of configuration that will be used to generate the proxy configuration.
+{{ site.mesh_product_name }} combines policies with dataplane configuration to generate the Envoy configuration of a proxy.
+
+## What do policies look like?
+
+Like all [resources](/docs/{{ page.version }}/introduction/concepts#resource) in {{ site.mesh_product_name }}, there are two parts to a policy: the metadata and the spec.
+
+### Metadata
+
+Metadata identifies the policies by its `name`, `type` and what `mesh` it's part of:
+
+{% tabs metadata %}
+{% tab metadata Universal %}
+
+```yaml
+type: ExamplePolicy
+name: my-policy-name
+mesh: default
+spec: ... # spec data specific to the policy kind
+```
+
+{% endtab %}
+{% tab metadata Kubernetes %}
+
+In Kubernetes all our policies are implemented as [custom resource definitions (CRD)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) in the group `kuma.io/v1alpha1`.
+
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: ExamplePolicy
+metadata:
+  name: my-policy-name
+  namespace: {{ site.mesh_namespace }}
+spec: ... # spec data specific to the policy kind
+```
+
+By default the policy is created in the `default` mesh.
+You can specify the mesh by using the `kuma.io/mesh` label.
+
+For example:
+
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: ExamplePolicy
+metadata:
+  name: my-policy-name
+  namespace: {{ site.mesh_namespace }}
+  labels:
+    kuma.io/mesh: "my-mesh"
+spec: ... # spec data specific to the policy kind
+```
+
+{% warning %}
+Policies are namespace-scoped and currently the namespace must be the one the control-plane is running in `{{site.mesh_namespace}}` by default.
```

Comment:
this warning is not true. In 2.8 and 2.9 you can specify different namespace than system namespace